db/models.js

```diff
@@ -0,0 +1,11 @@
+const db = require('./connection.js');
+const models={};
+const {articleData, commentData, topicData, userData}   = require('../db/data/test-data');
+
+models.fetchTopics = () => {
```

Comment:
Same comment from the controllers file applies here about this models object you're creating. Let me know if it doesn't make sense